ios/Classes/BetterPlayer.m

```diff
@@ -830,6 +847,41 @@ - (void)setMixWithOthers:(bool)mixWithOthers {
   }
 }
 
+- (void) initBlackCoverView {
+    _blackCoverView = NULL;
+    _blackCoverView = [[UIView alloc] init];
+    _blackCoverView.translatesAutoresizingMaskIntoConstraints = false;
+    _blackCoverView.backgroundColor = [UIColor blackColor];
+}
+
+- (void) showBlackCoverView {
+    UIWindow *window = [UIApplication sharedApplication].windows.firstObject;
+    if (window) {
+        [window addSubview:_blackCoverView];
+        // Set up constraints to make the child view fill its parent
+        NSLayoutConstraint *topConstraint = [_blackCoverView.topAnchor constraintEqualToAnchor:window.topAnchor];
+        NSLayoutConstraint *bottomConstraint = [_blackCoverView.bottomAnchor constraintEqualToAnchor:window.bottomAnchor];
+        NSLayoutConstraint *leadingConstraint = [_blackCoverView.leadingAnchor constraintEqualToAnchor:window.leadingAnchor];
+        NSLayoutConstraint *trailingConstraint = [_blackCoverView.trailingAnchor constraintEqualToAnchor:window.trailingAnchor];
+
+        // Activate the constraints
+        [topConstraint setActive:YES];
+        [bottomConstraint setActive:YES];
+        [leadingConstraint setActive:YES];
+        [trailingConstraint setActive:YES];
```

Comment:
Because we set active all constraints, so there is a short way to write this (we don't need create any variables)
```
[NSLayoutConstraint activateConstraints:@[
   [_blackCoverView.topAnchor constraintEqualToAnchor:window.topAnchor],
   ...
]];
```